packages/frontend/src/components/message/MessageMetaData.tsx

```diff
@@ -50,6 +50,8 @@ export default function MessageMetaData(props: Props) {
         ),
       })}
     >
+      {/* TODO a11y: add `aria-live` in case a padlock gets removed?
+      e.g. when a message gets edited, but the edit is not encrypted. */}
```

Comment:
not sure if you can even send an unencrypted edit to an encrypted message. I don't think this todo is necessary.

CC @r10s

Reply:
yip, it is not possible to send unencrypted edit to encrypted message. so, padlock cannot be removed by edits

Reply:
Thanks for checking this out!